lib/fork.js

```diff
@@ -18,6 +18,10 @@ module.exports = function (args) {
 
 	var ps = childProcess.fork(babel, args, options);
 
+	function send(command, data) {
+		ps.send({'ava-child-process-command': command, 'data': data});
```

Comment:
My only thought there was wether or not we wanted to monkeypatch `process.on` on the child process, so users never saw our communication.

I can imagine a scenario where you are writing some code that *might* be a child process (or might be top level), and so they would will listen to `process.on('message')`. By choosing a unique property name here we could monkeypatch `on` and filter out these messages.  I reemit these on the child process as specific `ava-` prefixed events(`process.on('ava-kill', ...)`, `process.on('ava-cleanup')`, etc) - that it is really unlikely users will listen to.

None of that is implemented. Just leaving the door open.